docs/articles/commands/intro.md

```diff
@@ -1,221 +1,328 @@
+using DSharpPlus.CommandsNext.Attributes;
 
-namespace MyFirstBot
+public class MyFirstModule : BaseCommandModule
 {
-    class MyCommands
+    [Command("greet")]
+    public async Task GreetCommand(CommandContext ctx)
     {
+        await ctx.RespondAsync("Greetings! Thank you for executing me!");
     }
 }
 ```
 
-Add a `public` modifier to the class. That class will now serve as your command module.
+### Cleanup and Configuration 
+Before we can run our new command, we'll need modify our main method.<br/>
+Start by removing the event handler we created [previously](xref:basics_first_bot#spicing-up-your-bot).
+```cs
+var discord = new DiscordClient();
 
-Before you can proceed, add `using System.Threading.Tasks;`, `using DSharpPlus;`, 
-`using DSharpPlus.CommandsNext;`, and `using DSharpPlus.CommandsNext.Attributes;` to the using section.
+discord.MessageCreated += async (s, e) =>               // REMOVE
+{                                                       // ALL
+    if (e.Message.Content.ToLower().StartsWith("ping")) // OF
+        await e.Message.RespondAsync("pong!");          // THESE
+};                                                      // LINES
+
+await discord.ConnectAsync();            
+```
 
-Go back to your main bot class, and below the command module initialisation, add the following:
+<br/>
+Next, call the `UseCommandsNext` extension method on your `DiscordClient` instance and pass it a new `CommandsNextConfiguration` instance.
+Assign the resulting `CommandsNextExtension` instance to a new variable named *commands*. This important step will enable CommandsNext for your Discord client.
+```cs
+var discord = new DiscordClient();
+var commands = discord.UseCommandsNext(new CommandsNextConfiguration());
+```
 
+Create an object initializer for `CommandsNextConfiguration` and assign the `StringPrefixes` property a new `string` array containing your desired prefixes.
+Our example below will only define a single prefix: `!`.
 ```cs
-commands.RegisterCommands<MyCommands>();
+new CommandsNextConfiguration()
+{ 
+    StringPrefixes = new[] { "!" }
+}
 ```
 
-This will enable all the commands in your command module.
+<br/>
+Now we'll register our command module.
+Call the `RegisterCommands` method on our `CommandsNextExtension` instance and provide it with your command module.
+```cs
+var discord = new DiscordClient();
+var commands = discord.UseCommandsNext();
 
-The class should now look like this:
+commands.RegisterCommands<MyFirstModule>();
 
+await discord.ConnectAsync();            
+```
+Alternatively, you can pass in your assembly to register commands from all modules in your program.
+```cs
+commands.RegisterCommands(Assembly.GetExecutingAssembly());
+```
+
+<br/>
+Your main method should look similar to the following:
 ```cs
-using System;
 using System.Threading.Tasks;
 using DSharpPlus;
 using DSharpPlus.CommandsNext;
+using MyFirstBot.Commands;
 
-namespace MyFirstBot
+internal static async Task MainAsync()
```

Comment:
Looking at this from the doc view, it appears this method doesn't have a class or a namespace, which is a syntax error. Either the class and namespace should be re-added or the using statements should be removed if you just want to focus on that method.